cmake.deps/CMakeLists.txt

```diff
@@ -132,6 +132,18 @@ if(CMAKE_OSX_ARCHITECTURES)
   endforeach()
 endif()
 
+# If the macOS deployment target is not set manually (via $MACOSX_DEPLOYMENT_TARGET),
+# fall back to local system version. Needs to be done here and in top-level CMakeLists.txt.
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  if(NOT CMAKE_OSX_DEPLOYMENT_TARGET)
```

Comment:
When do you get here and `CMAKE_OSX_DEPLOYMENT_TARGET` is not set? The docs make it sound like it should be set when you call `project()`, but I'm guessing that's not true...

Reply:
I consider CMake to have a realdokumentation: it's documented by how it behaves in practice.